app/javascript/styles/_scatterPlot.scss

```diff
@@ -63,6 +66,21 @@
 .scatter-legend-entry {
   position: relative;
   top: 5px;
+  overflow: scroll;
```

Comment:
This has the same problem with `overflow` as the other PR - this should be set to `auto`.

Reply:
[Done over in the base branch](https://github.com/broadinstitute/single_cell_portal_core/pull/1240#discussion_r744824672).

Reply:
[Done over in the base branch](https://github.com/broadinstitute/single_cell_portal_core/pull/1240#discussion_r744824672).